documentation/asciidoc/topics/proc_configuring_pod_termination_grace_period.adoc

```diff
@@ -0,0 +1,21 @@
+[id='configuring-pod-termination-grace-period{context}']
```

Comment:
@pminz I was about to merge and then I realised we forgot the documentation. I have pushed these real quick. Please review and if you think we have missed something we can address in a future PR